tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "esModuleInterop": true,
     "experimentalDecorators": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["es6"],
+    "lib": ["es2020"],
```

Comment:
I would propagate this change to the rest of the solidity repositories. Could you please manage it opening the needed PR's?